dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -1,17 +1,14 @@
 - level ||= @level
-- last_attempt ||= @last_attempt
+- last_attempt = @last_attempt unless local_assigns.has_key? :last_attempt
 - index ||= nil
 
 - if index # If an index was provided we're inside a LevelGroup.
   :javascript
-    window['level_#{index}'] = {
+    window.levelGroup.questions[#{level.id}] = {
```

Comment:
Hm, I was thinking levels (since a levelgroup is made out of levels in most places)... or is that name too overloaded?